src/main/java/racingGame/model/Car.java

```diff
@@ -1,24 +1,68 @@
 package racingGame.model;
 
+import java.util.Objects;
+
 public class Car {
     
-    public static final int INIT_LOCATION = 1;
+    public static final int INIT_LOCATION = 0;
     public static final int CAR_FORWARD_CRITERIA = 3;
+    private final String name;
     private int location;
     
-    public Car() {
+    public Car(Car car) {
+        this(car.name, car.location);
+    }
+    public Car(String name, int location) {
+        this(name);
+        this.location = location;
+    }
+
+    public Car(String name) {
+        validate(name);
+        this.name = name;
         this.location = INIT_LOCATION;
+    }    
```

Comment:
```suggestion
    public Car(String name) {
        this(name, INIT_LOCATION);
    }    
    
    public Car(String name, int location) {
       validate(name);
        this.name = name;
        this.location = location;
    }
```
모든 유효성 체크, 필드 값 초기화는 주 생성자에서..
부 생성자는 주 생성자를 호출